client/spring-zeebe-starter/src/main/java/io/zeebe/spring/client/actuator/ZeebeClientHealthIndicator.java

```diff
@@ -0,0 +1,26 @@
+package io.zeebe.spring.client.actuator;
+
+import io.zeebe.client.ZeebeClient;
+import io.zeebe.client.api.response.Topology;
+import org.springframework.boot.actuate.health.AbstractHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+
+public class ZeebeClientHealthIndicator extends AbstractHealthIndicator {
+
+  private final ZeebeClient client;
+
+  public ZeebeClientHealthIndicator(ZeebeClient client) {
+    this.client = client;
+  }
+
+  @Override
+  protected void doHealthCheck(Health.Builder builder) {
```

Comment:
Hey @pabloromanelli - thank you so much for this PR! Happy to merge it before I release the 0.23 for Spring :-) One quick question: Do you know what happens if the broker there is an exception raised in this method (as if the broker is really down)? does this count as unhealthy?

Reply:
Happy to help :)
It should count as unhealthy because in `AbstractHealthIndicator` they catch exceptions thrown by `doHealthCheck` to produce a "down" status:

```java
	public final Health health() {
		Health.Builder builder = new Health.Builder();
		try {
			doHealthCheck(builder);
		}
		catch (Exception ex) {
			if (this.logger.isWarnEnabled()) {
				String message = this.healthCheckFailedMessage.apply(ex);
				this.logger.warn(StringUtils.hasText(message) ? message : DEFAULT_MESSAGE, ex);
			}
			builder.down(ex);
		}
		return builder.build();
	}
```

Reply:
How it should and how I expected it. Great! Thanks!